docs/content/guides/limiting-concurrency-in-data-pipelines.mdx

```diff
@@ -255,23 +255,26 @@ assets_job = define_asset_job(
 Op-based jobs are defined using the <PyObject object="job" decorator /> decorator. In this example, using the <PyObject module="dagster" object="multiprocess_executor" />, the job will execute up to four ops at once:
 
 ```python file=/concepts/ops_jobs_graphs/job_execution.py startafter=start_tag_concurrency endbefore=end_tag_concurrency
+from dagster import job, RunConfig
+
+
 @job(
-    config={
-        "execution": {
+    config=RunConfig(
+        execution={
             "config": {
                 "multiprocess": {
-                    "max_concurrent": 4,
+                    "max_concurrent": 0,
```

Comment:
Same comment here as on job execution - this change puts the example at odds with the paragraph that comes before it.